### PR TITLE
Add static method to stop validator from validate

### PR DESCRIPTION
**Pull request by kopecmi8**
HTML validator validates all content which is put on standard output. Once you append it in your config file there is no posibility to stop them from validation on specific page, and this can be tricky.

For example if your presenter serves big files (>30MB) from your file system because of its gzip compression, your allowed memory size can be easily exhausted, because validator wants all the file content parse. 

This static method can solve this problem, and simply by calling  
`\Kdyby\Extension\Diagnostics\HtmlValidator\ValidatorPanel::stopValidate();`
you can temporary stop validator from validates output content.

### CODE REVIEW

File: ValidatorPanel.php [hunk 1]
```diff
@@ -863,28 +875,31 @@ public function validate()
...
-		$dom->preserveWhiteSpace = FALSE;
-		$dom->strictErrorChecking = TRUE;
-		$dom->recover = TRUE;
-
-		set_error_handler(function($severity, $message) {
+		
+		if(self::$validate) {
+			libxml_use_internal_errors(true);
+			$dom = new \DOMDocument('1.0', 'UTF-8');
+			$dom->resolveExternals = FALSE;
```
Comment (simPod) [inline ValidatorPanel.php]: can we use lower case for `FALSE`/`TRUE` in modified code?

File: ValidatorPanel.php [hunk 2]
```diff
@@ -853,6 +858,13 @@ public function stopBuffering()
 		if (ob_get_level()) ob_end_flush();
 	}
 
+	/**
+	 * Stop validator.
```
Comment (simPod) [inline ValidatorPanel.php]: Useless comment

File: ValidatorPanel.php [hunk 3]
```diff
@@ -853,6 +858,13 @@ public function stopBuffering()
...
+	 */
+	public static function stopValidate()
```
Comment (simPod) [inline ValidatorPanel.php]: Stop validate IMHO means to stop something that is already running. What about `disableValidation()`?

**Comment by kopecmi8** [comment]
You are right. It was wrong idea, now I doing this better as you suggest in config file. I created another pull request because it is a whole another solution. Follow this https://github.com/Kdyby/HtmlValidatorPanel/pull/10

**Pull request closed without merging**